src/mono/wasm/debugger/DebuggerTestSuite/BreakpointTests.cs

```diff
@@ -811,5 +819,66 @@ public async Task CreateGoodBreakpointAndHitGoToWasmPageWithoutAssetsComeBackAnd
                 }
             );
         }
+
+        [Fact]
+        public async Task BreakInDefaultInterfaceMethod()
+        {
+            string methodName = "DefaultMethod";
+            string assembly = "dotnet://debugger-test.dll/debugger-test.cs";
+            var bp_dim = await SetBreakpointInMethod("debugger-test.dll", "IDefaultInterface", methodName, 1);
+            var init_location = await EvaluateAndCheck(
+                $"window.setTimeout(function() {{ invoke_static_method('[debugger-test] DefaultInterfaceMethod:Evaluate'); }}, 1);",
+                assembly,
+                bp_dim.Value["locations"][0]["lineNumber"].Value<int>(),
+                bp_dim.Value["locations"][0]["columnNumber"].Value<int>(),
+                methodName,
+                wait_for_event_fn: async (pause_location) => { await CheckDefaultMethod(pause_location); }
+            );
+            await SendCommandAndCheck(null, "Debugger.resume",
```

Comment:
What is this second check for? IIUC, it is only telling us that we stopped in the DIM again.

Reply:
> My idea was to check that we are able to control debugger correctly in DIM, not only hit bp there but also resume from inside.

Good thinking. Mention that in a comment above the call.